examples/tutorials/3d-map.py

```diff
@@ -0,0 +1,140 @@
+"""
+Creating a 3D map
+=================
+
+Plotting a three-dimensional map is handled by :meth:`pygmt.Figure.grdview`.
+"""
+
+import pygmt
+
+# Load sample earth relief data
+grid = pygmt.datasets.load_earth_relief(resolution="05m", region=[-108, -103, 35, 40])
+
+########################################################################################
+# The :meth:`pygmt.Figure.grdview` method takes the ``grid`` input.
+# The ``perspective`` argument changes the azimuth and angle of the viewpoint; the
+# default is [180, 90], which is looking directly down on the figure and north is "up".
+# The ``zsize`` argument sets how tall the three-dimensional portion appears.
+#
+# The default figure surface is *mesh plot*.
```

Comment:
This sentence doesn't read well to me. Perhaps change to "The default grid surface type is *mesh plot*"?

Reply:
Changed to your suggestion.